src/gui/Src/Gui/ShortcutsDialog.h

```diff
@@ -28,10 +28,14 @@ private slots:
     void on_btnSave_clicked();
     void on_btnClearShortcut_clicked();
     void rejectedSlot();
-
+    void on_btnClearFilter();
+    void on_FilterTextChanged(const QString & actionName);
 private:
     Ui::ShortcutsDialog* ui;
     QMap<QString, Configuration::Shortcut> ShortcutsBackup;
+    void filterShortcutsByName(const QString & nameFilter, QMap<QString, Configuration::Shortcut> & mapToFill);
+    void renderShortcuts(QMap<QString, Configuration::Shortcut> & shortcuts);

```

Comment:
This function is not rendering anything, it is also only used in `showShortcutsFiltered`, so inlining it there would be better I think.